src/test/java/com/demoqa/shop/hooks/Hooks.java

```diff
@@ -1,4 +1,43 @@
 package com.demoqa.shop.hooks;
 
+import com.demoqa.shop.pages.HomePage;
+import com.demoqa.shop.util.Browser;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestInfo;
+import org.openqa.selenium.WebDriver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public class Hooks {
+
+
+    Logger log = LoggerFactory.getLogger(Hooks.class);
+
+    @Before(order =1)
+    public void beforeScenario(){
+
+        WebDriver driver = Browser.getBrowser();
+
+        HomePage home = new HomePage(driver);
+        home.lnk_Dismiss.click();
+    }
+
+    @After(order =1)
+    public void afterScenario(){
+
```

Comment:
maybe a browser.close() could be added here. Not sure if it is added anywhere else

Reply:
Exactly. I was waiting Browser implementation and discussing with team about implementation.